tests/scripts/parse_notice.py

```diff
@@ -0,0 +1,277 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+
+import json
+import re
+import sys
+from datetime import datetime
+from typing import Any
+
+import requests
+from requests import Response
+
+
+class NoticeParser:
+    POSSIBLE_LICENSE_FILES: list[str] = [
+        "LICENSE",
+        "LICENSE.txt",
+        "LICENSE.rst",
+        "apache-2.0.LICENSE",
+        "apache-2.0.LICENSE.txt",
```

Comment:
this list seems a little short, or is it only my impression? :)

Reply:
Usually doesn't matter what type of license it is (MIT, apache, etc.) they're usually found in `LICENSE.txt` or `LICENSE`, but people may use other file names for those like "apache-2.0.License" etc.
These are the files I've encountered so far, but I'll add more possible combinations.